public/app/user-service-registry/user-service-registry.tpl.html

```diff
@@ -295,6 +301,12 @@ <h1 class="alert alert-info">Inputs</h1>
                     <textarea ng-model="textInput.content" rows="10" cols="80">
                         </textarea>
                 </p>
+                <p>
+                <label>Select Format MimeType</label>
+                <select name="formatSelect" id="bodyInputFormatSelect" ng-model="textInput.formatSelect">
```

Comment:
maybe change this ng-model to bodyInput... angular uses ng-model for data binding and i'm not sure how it does when there are multiple models with the same name